x-pack/plugins/graph/public/plugin.ts

```diff
@@ -44,15 +62,42 @@ export class GraphPlugin implements Plugin<{ config: Readonly<ConfigSchema> }, v
       });
     }
 
-    return {
-      /**
-       * The configuration is temporarily exposed to allow the legacy graph plugin to consume
-       * the setting. Once the graph plugin is migrated completely, this will become an implementation
-       * detail.
-       * @deprecated
-       */
-      config: this.initializerContext.config.get(),
-    };
+    const config = this.initializerContext.config.get();
+
+    initAngularBootstrap();
+    core.application.register({
+      id: 'graph',
+      title: 'Graph',
+      order: 9000,
+      appRoute: '/app/graph',
+      euiIconType: 'graphApp',
+      category: DEFAULT_APP_CATEGORIES.analyze,
+      mount: async (params: AppMountParameters) => {
+        const [coreStart, pluginsStart] = await core.getStartServices();
+        const { renderApp } = await import('./application');
+        return renderApp({
+          ...params,
+          pluginInitializerContext: this.initializerContext,
+          licensing,
+          core: coreStart,
+          navigation: pluginsStart.navigation,
+          npData: pluginsStart.data,
```

Comment:
While we are at it, can you rename this service to just `data`?

Reply:
Sure, done.